osu.Game/Screens/Ranking/Expanded/Accuracy/AccuracyCircle.cs

```diff
@@ -63,7 +62,7 @@ public partial class AccuracyCircle : CompositeDrawable
         /// <summary>
         /// Relative width of the rank circles.
         /// </summary>
-        public const float RANK_CIRCLE_RADIUS = 0.06f;
+        public const float RANK_CIRCLE_RADIUS = 0.05f;
```

Comment:
Are these constant changes to account for the sharper display without buffered container, or something else?

Reply:
> Are these constant changes to account for the sharper display without buffered container

Basically yes. Without `BufferedContainer` they became a tiny bit thicker, so I'm just accounting for that. Used master as a visual reference. Can upload an image comparison mster/old values/new values